tutorials/ph-mult-rasters/index.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Using Earth Engine to export multiple rasters based on user-defined dates
```

Comment:
You can remove "Using Earth Engine to.." since the tutorial will be under Earth Engine Tutorials section of the docs.

Reply:
Done